Makefile.common

```diff
@@ -110,13 +112,17 @@ ARCH =  $(SIL)@mkdir -p $(@D); \
 	$(AR) $(ARFLAGS) $@ $^ 2>/dev/null
 
 LINK =  $(SIL)mkdir -p $(@D); \
-	(printf "LD  %-24s\n" "$(@F)"); \
+	(printf "LD  %-24s\n" "$(@:$(PREFIX_BUILD)/%=%)"); \
 	$(LD) $(LDFLAGS) -o "$@"  $^ $(LDLIBS)
 	
 HEADER = $(SIL)mkdir -p $(@D); \
 	(printf "HEADER %-24s\n" "$<"); \
 	cp -pR "$<" "$@"
 
+INSTALL_FS = $(SIL)(printf "INSTALL_FS %s\n" "$(@:$(PREFIX_FS)%=%)"); \
+	     mkdir -p "$(dir $@)"; \
+	     install -m 755 "$<" "$(dir $@)"
+
 $(PREFIX_O)%.o: %.c
 	@mkdir -p $(@D)
 	$(SIL)(printf "CC  %-24s\n" "$<")
```

Comment:
```suggestion
	$(SIL)printf "CC  %-24s\n" "$<"
```
as above

Reply:
not related to this PR, so leaving as-is. I agree that the parentheses are not needed in this case.